### DISCUSSION TITLE
case-studies/the-future-of-everything/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#214] [Website Discussions]
# case-studies/the-future-of-everything/

Animated in Animate, Finalised in Final Cut

https://fcp.cafe/case-studies/the-future-of-everything/

**Comment by florian-duffe** [comment]
So cool. Thanks, Kevin.
The timeline is looking very lean. Have you ever worked with Compound Clips, stacking different versions of shots, or did you immediately switch it with the newest version in the primary storyline. Could you make use of effects, generators or color grading to beef up the animation?

**Comment by kevinluk07** [comment]
Thanks Florian!  In most case, I simply duplicate the entire project, and then I'll put the new version on top. The snapping wil tell me if I have trimmed the shot, and then overwrite it to the primary storyline. Using compound clip will definitely take out the hassle of comparing length and copying over any effects, but I kept most shot as is when I'm expecting major animation revisions, and effects are also done near the end of the editing proces, so I find it more effective managing it all with project versions instead of snapshots and compound clips.

Most of the effects on the show where done in the animation stage. There were a fair bit of glow and a kaleidoscope effect that was done in *A Tougher Power* and *Long Term Growth* respectively. These shots comes to me as both the Animate file as well as an exported Quicktime made in After Effects, I believe. *Unlimited Ducks* was the exception, with two style departure scenes for the seagull uprising and the duck steak. That two scenes were the ones that I have applied compound clips and then some grade and effects. All of the epsiode, however, has a grain and a slight blur added to it with an adjustment layer, and solid colour made with the custom generator for the title cards.

![Effects in the Series](https://github.com/CommandPost/FCPCafe/assets/13616091/3694a37f-026a-47b6-a9c0-04e0480b976e)

If you look at the glitches of the ducks around the one minute mark in the episode *Unlimited Ducks*, you would see that the black outlines are clearly drawn on the glitchy blocks. That is something a bit trickier to do in Final Cut so it was done during animation. But the duck-shaped transition and the film projection transition in the same episode doesn't affect the outlines and were done in Final Cut. The film transition is technically breaking the art style, but in a comedy animation where we are doing style departures anyway, I think it fits well!

![Effects in the Episode Unlimited Ducks](https://github.com/CommandPost/FCPCafe/assets/13616091/f8c4dee8-c29e-4e64-a7a9-daf02f5167a3)
